packages/datagateway-dataview/src/views/table/datafileTable.component.tsx

```diff
@@ -161,6 +161,17 @@ const DatafileTable = (props: DatafileTableProps): React.ReactElement => {
     [cartItems, selectAllSetting, allIds]
   );
 
+  const isParentSelected = React.useMemo(() => {
```

Comment:
Could be tidied up by having a single `.some` or `.includes` rather than checking both separately with OR ...